index.html

```diff
@@ -0,0 +1,1376 @@
+                        </path>
+                    </g>
+                </svg>
+
+            </a>
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarTogglerDemo02"
+                aria-controls="navbarTogglerDemo02" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+
+            <div class="collapse navbar-collapse" id="navbarTogglerDemo02">
+                <ul class="navbar-nav mr-auto mt-4 pt-2 mt-lg-0">
+                    <li class="nav-item active">
+                        <a class="nav-link font-weight-bold section-heading text-white" href="#">U.S <span
+                                class="sr-only">(current)</span></a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link section-heading font-weight-bold text-white" href="#">World</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link section-heading font-weight-bold text-white" href="#">World</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link section-heading font-weight-bold text-white" href="#">Business</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link section-heading font-weight-bold text-white" href="#">Tech & Science</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link section-heading font-weight-bold text-white" href="#">Culture</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link section-heading font-weight-bold text-white" href="#">Sports</a>
+                    </li>
+                </ul>
+                <div>
+                    <p class="text-right section-heading font-weight-bold mt-2 mr-3 pt-3"><a href="#"
+                            class="text-white">SIGN
+                            IN</a></p>
+                </div>
+                <div class="btn-light  bg-danger text-left pt-2">
+                    <button class="subscribe-boot section-heading pr-2 font-weight-bold">SUBSCRIBE ></button>
+                </div>
+
+            </div>
+        </nav>
+
+        <!--END OF NAV-->
+
+        <!--START OF FEATURED STORIES-->
+        <div class="container-fluid header-section">
+            <div class="row border-top border-muted">
+                <section class="col-xl-4 col-lg-4 col-md-5 col-sm-12 mt-4 order-sm-second order-md-first pl-lg-4">
```

Comment:
featured stories section is wrapped around `<section>`, and each story inside it has `<article>` tag.